### ISSUE TITLE
WS connection breaks on high load on backend side

### ISSUE THREAD

**Issue by IoTMOD** [iotaledger/hornet#50]
**Describe the bug**
The WS connection breaks if there is high load on the Backend side (HORNET syncing).
After a page refresh it's working again until it breaks again.

**To Reproduce**
- Delete the mainnetdb folder
- Start HORNET & open the dashboard
- Dashboard WS connection will break

**Environment information:**
 - OS: Ubuntu 18.04
 - RAM:  4 GB
 - Cores: 2 Cores
 - Type: VPS
 - Browser: Chrome
 - HORNET version: 0.2.7

**Additional context**
```
[ ERROR ] SPA: error while writing to web socket client xxxx:xxxx:xxxx:xxxx:xxxx:xxxx:xxx:xxxx: write tcp [xxxx:xxx:xxx:xxxx::1]:8081->[xxxx:xxxx:xxxx:xxxx:xxxx:xxxx:xxx:xxxx]:59344: i/o timeout
```

**Comment by muXxer** [comment]
Even worse, I'm just a little bunny

Luca, wen not edit my comments? :rage:

**Comment by muXxer** [comment]
Should be fixed with v0.2.8
